toolchain/mfc/user_guide.py

```
@@ -0,0 +1,649 @@
+            "[bold yellow]Troubleshooting Tips[/bold yellow]\n\n"
+            "  [cyan]1.[/cyan] Run with [green]--debug-log[/green] to see detailed output\n"
+            "  [cyan]2.[/cyan] Check [green]docs/documentation/troubleshooting.md[/green]\n"
+            "  [cyan]3.[/cyan] Ensure required modules are loaded: [green]source ./mfc.sh load -c <cluster> -m <mode>[/green]\n"
+            "  [cyan]4.[/cyan] Try [green]./mfc.sh clean[/green] and rebuild",
+            box=box.ROUNDED,
+            border_style="yellow",
+            padding=(0, 2)
+        ))
+
+    @staticmethod
+    def after_case_error(case_path: str = None):
```

Ruff (0.14.14)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)